woocommerce/woocommerce-android
# Issue/6360 canada fee set manually

Closes: #6360 


### Description

The PR adds a fee calculation on the client-side. (Canada only) The formula is the following:

```
15 cents flat 
```

### Testing instructions

* Make a payment in the US and check stripe dashboard, notice that the fee is calculated by 2.6% + $0.10
* Make a payment in the CA (interac) and check stripe dashboard, notice that the fee is calculated by $0.15 CA
* Make a payment in the CA (not Interac) and check stripe dashboard, notice that the fee is calculated by 2.6% + $0.10 CA (WCpay overrides the client side value we set)

(still not clear but apparently it has to be `2.7 + 5 cents` but it's on the backed so we are good to proceed here)



~~UPD: added do not merge because still not clear what fee is for the interac payments~~
